types/spectest/tests/consensusdata/sync_committee_contrib_no_justifications.go

```diff
@@ -0,0 +1,24 @@
+package consensusdata
+
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// SyncCommitteeContributionNoJustifications tests an invalid consensus data with no sync committee contribution pre-consensus justifications
+func SyncCommitteeContributionNoJustifications() *ConsensusDataTest {
+
+	// To-do: add error when pre-consensus justification check is added.
```

Comment:
is this a test todo or we missing this check in general?

Reply:
when we bring back pre-consensus justification it shouldn't be empty